docker/default.env

```diff
@@ -0,0 +1,38 @@
+# Default configuration for Docker containers
+# To modify, copy these over to a ".env" file
+# and values here will be overridden.
+#
+# Values passed via command-line arguments take precedence over .env files.
+#
+# Note that there is no special handling of quotation marks.
+# This means that they are part of the value.
+#
+# Note that these variables are not available at docker-compose.yml
+# Variables show up defined only inside containers.
+
+# WordPress
+WP_DOMAIN=localhost
+WP_ADMIN_USER=wordpress
+WP_ADMIN_EMAIL=wordpress@example.com
+WP_ADMIN_PASSWORD=wordpress
+WP_TITLE=HelloWord
+
+# Database
+MYSQL_HOST=db:3306
+MYSQL_DATABASE=wordpress
+MYSQL_USER=wordpress
+MYSQL_PASSWORD=wordpress
+MYSQL_ROOT_PASSWORD=wordpress
```

Comment:
These variables are what mysql Docker container expects to have.

See _"Environment Variables"_ from https://hub.docker.com/_/mysql/